src/compiler/transformers/module/module.ts

```diff
@@ -1662,6 +1668,28 @@ namespace ts {
             return node;
         }
 
+        function substituteCallExpression(node: CallExpression) {
+            if (!isIdentifier(node.expression)) {
+                return node;
+            }
+            const newExpression = substituteExpressionIdentifier(node.expression);

```

Comment:
The substitution for the identifier should be invoked by the printer, you shouldn't have to call it directly.